mandos-go/json/integrationTests/example.scen.json

```diff
@@ -257,6 +257,7 @@
             "step": "scCall",
             "txId": "1c",
             "comment": "without logs",
+            "displayLogs": true,
```

Comment:
comment: without logs
displayLogs: true 

:thinking:

Reply:
not my fault that events are translated to logs.

Reply:
moved test to other step